web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -40,7 +41,7 @@ const getLicenseFromMd = (licenseMd) => {
     "department-phone-2": licenseMd.divisionPhone,
     "license-certification-classification": licenseMd.licenseCertificationClassification,
     "form-name": licenseMd.formName,
-    "primary-industry": licenseMd.industryId ?? licenseMd.webflowIndustry,
+    "primary-industry": LookupIndustryById(licenseMd.industryId).name ?? licenseMd.webflowIndustry,
```

Comment:
I don't think `??` is the right thing here. `LookupIndustryById` returns an object with `name` set to `""` if the id passed is undefined.

Empty string doesn't count as undefined, so it would not actually cascade to the `webflowIndustry` part here